io-package.json

```diff
@@ -175,13 +175,17 @@
         "readme": "https://github.com/ioBroker/ioBroker.node-red/blob/master/README.md",
         "stopBeforeUpdate": true,
         "adminTab": {
-            "link": "http://%ip%:%port%%httpRoot%",
+            "link": "http%s%://%ip%:%port%%httpRoot%",
             "fa-icon": "settings_input_composite"
         }
     },
     "native": {
         "bind": "0.0.0.0",
         "port": 1880,
+        "secure": false,
+        "s": "",
```

Comment:
?? Whats this?

Reply:
To my understanding all instance settings of an iobroker adapter (specified in index_m.html) get a default value here. (e.g. port: 1880)
I added new parameters for HTTPS:
* secure: HTTPS enabled or not
* s: setting for admin-link to node-red. If HTTPS is enabled, an "s" will be stored in the setting and added to the url (http:// -> https:// )
* certPublic: selected public key of certificate
* certPrivate: selected private key of certificate

Reply:
I know, it is not logical, but there is a special replace: https://github.com/ioBroker/ioBroker.admin/blob/master/src/js/adminInstances.js#L40
So 'protocol' will be replaced with 'http' or 'https' automatically.